languages/en.ini

```diff
@@ -1090,6 +1091,13 @@ User Account = "User Account"
 Username = "Username"
 Username cannot be blank = "Username cannot be blank"
 Username is already in use in another library card = "Username is already in use in another library card"
+verification_done = "Your email address has been verified successfully."
+verification_email_notification = "A request was just made to verify your email address for your account with %%library%%."
+verification_email_sent = "Email address verification instructions have been sent to the email address registered with this account."
+verification_email_subject = "VuFind Email Verification"
+verification_email_url_pretext = "You can verify your email address at this URL: %%url%%"
+verification_too_soon = "Too many email address verification requests have been made, please try again later"
```

Comment:
It appears that an earlier comment I left here got lost. Apologies if I'm accidentally submitting a duplicate comment.

I think the language on this message may be too strong. If you try to log in immediately after registering, you will see this message, and I'm not sure that it should be treated as an error. Would it be better to say something like "Your email requires validation. An email was recently sent to your registered email address. If you did not receive it, please wait a few minutes and try logging in again, and a new email will be sent."